### PR TITLE
Svelte: Update documentation for current best practices

### PR DESCRIPTION
**Pull request by ObieMunoz**
The original Testing Library/Svelte docs have not beed updated in quite a while and I believe the actual examples were written prior to `screen` being released in [DOM Testing Library v6.11.0](https://github.com/testing-library/dom-testing-library/releases/tag/v6.11.0). 

Best practice is to use the `screen` and not access the `container` directly for queries.

### CODE REVIEW

File: docs/queries/about.mdx [hunk 1]
```diff
@@ -223,6 +224,15 @@ const exampleInput = screen.getByLabelText('Example')
...
 cy.findByLabelText('Example').should('exist')
+```
+
+  </TabItem>
+  <TabItem value="svelte">
+
+```js
+import {render, screen} from '@testing-library/svelte'
+
+const exampleInput = screen.getByLabelText('Example')
```
Comment (timdeschryver) [inline docs/queries/about.mdx:235]: Should this render the component first?
I would also prefer to revert this change to keep this consistent, or all the examples in the core api should include a svelte example. If you want to, feel free to create a follow-up PR with the changes.

File: docs/svelte-testing-library/api.mdx [hunk 2]
```diff
@@ -30,6 +31,32 @@ import {render} from '@testing-library/svelte'
 const {results} = render(YourComponent, {ComponentOptions}, {RenderOptions})
 ```
 
+## `screen`
```
Comment (timdeschryver) [inline docs/svelte-testing-library/api.mdx:34]: To keep the docs consistent we should also revert this change because no other libraries include it in their docs.
`screen` is not Svelte specific and is already documented in the core api, ideally we don't want to introduce duplicates.

File: docs/svelte-testing-library/api.mdx [hunk 3]
```diff
@@ -128,8 +155,8 @@ import {render, fireEvent} from '@testing-library/svelte'
 import Comp from '..'
 
 test('count increments when button is clicked', async () => {
-  const {getByText} = render(Comp)
-  const button = getByText('Count is 0')
+  render(Comp)
+  const button = screen.getByText('Count is 0')
```
Comment (timdeschryver) [inline docs/svelte-testing-library/api.mdx:159]: `screen` needs to be imported

**Review by timdeschryver — CHANGES_REQUESTED** [review]
Hi thanks for the PR.
I added some remarks to keep things consistent.
Let me know what you think

File: docs/queries/about.mdx (hunk 1 above)
Comment (ObieMunoz) [reply docs/queries/about.mdx]: I reverted this and will follow-up with a future PR to add svelte to all of the examples in the core api.

File: docs/svelte-testing-library/api.mdx (hunk 2 above)
Comment (ObieMunoz) [reply docs/svelte-testing-library/api.mdx]: That makes sense, I reverted this.

File: docs/svelte-testing-library/api.mdx (hunk 3 above)
Comment (ObieMunoz) [reply docs/svelte-testing-library/api.mdx:132]: Thanks, updated!

**Review by timdeschryver — APPROVED** [review]
Thanks @ObieMunoz !

**Comment by timdeschryver** [comment]
@all-contributors please add @ObieMunoz  for docs
